orangecontrib/imageanalytics/widgets/owimageembedding.py

```diff
@@ -199,11 +199,14 @@ def commit(self):
 
         file_paths_attr = self._image_attributes[self.cb_image_attr_current_id]
         file_paths = self._input_data[:, file_paths_attr].metas.flatten()
+        origin = file_paths_attr.attributes.get("origin", "")
+
         assert file_paths_attr.is_string
         assert file_paths.dtype == np.dtype('O')
 
         file_paths_mask = file_paths == file_paths_attr.Unknown
         file_paths_valid = file_paths[~file_paths_mask]
+        file_paths_valid = origin + "/" + file_paths_valid
```

Comment:
Paths should be joined with [os.path.join](https://docs.python.org/3/library/os.path.html#os.path.join)

For example, when `origin==""` the current code would prepend a "/" to the path.

Reply:
Thanks, corrected.